Anomaly/Patches/ThingDefs_Misc/Apparel_Packs.xml

```diff
@@ -35,34 +34,122 @@
 		</value>
 	</Operation>
 
-  <!--Projectile-->
-  
-  <Operation Class="PatchOperationReplace">
-    <xpath>Defs/ThingDef[defName="Grenade_DisruptorFlare"]/thingClass</xpath>
-    <value>
-      <thingClass>CombatExtended.ProjectileCE_SpawnsThing</thingClass>
-    </value>
-  </Operation>
+	<!--Projectile-->
+	
+	<Operation Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Grenade_DisruptorFlare"]/thingClass</xpath>
+		<value>
+			<thingClass>CombatExtended.ProjectileCE_SpawnsThing</thingClass>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Grenade_DisruptorFlare"]/projectile</xpath>
+		<value>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<dropsCasings>true</dropsCasings>
```

Comment:
No need for casings as it's not a grenade, looks pretty much like a stick flare.